packages/jest-config/src/__tests__/getMaxWorkers.test.ts

```diff
@@ -32,7 +32,7 @@ describe('getMaxWorkers', () => {
 
   it('Returns based on the number of cpus', () => {
     expect(getMaxWorkers({})).toBe(3);
-    expect(getMaxWorkers({watch: true})).toBe(2);
+    expect(getMaxWorkers({watch: true})).toBe(3);
```

Comment:
was there any reason this was 1 less? @rogeliog do you know?

Reply:
Maybe the rationale for cpus/2 was that watch mode is typically used while doing other things (mostly editing files) and this was to make it less likely that the editor becomes slow / freezes?

Reply:
If we want to make it 1 less that's fine, but before it was _halved_! Just happened to be 1 less in this case.

On my machine, it was running 6 workers instead of 11.

Reply:
I know, stumbled over the formula as well some time ago and wondered if there's any experimental basis for it ^^
I'm fine with using `cpus - 1` for watch mode as well. It's a good heuristic for what performs best when Jest is the only thing running, we can't really do much about sharing resources with other applications.
If you want your editor to remain responsive while running an expensive task, `nice yarn jest --watch` is a good idea anyway.